opentelemetry/src/global/internal_logging.rs

```diff
@@ -25,7 +28,13 @@ macro_rules! otel_info {
         {
             tracing::info!( name: $name, target: env!("CARGO_PKG_NAME"), name = $name, "");
         }
-        #[cfg(not(feature = "internal-logs"))]
+
+        #[cfg(test)]
```

Comment:
this is a good idea. Let us do this in a separate PR, as there are some tests which setup fmt layer already, and we need to discontinue that to avoid the duplication to stdout.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/2841 opened to check this.

Reply:
@cijothomas great - thanks! I'll chase this up.